src/test/java/harness/TestUser.java

```diff
@@ -89,8 +89,10 @@ public void login() throws IOException {
     writeTerraOAuthCredentialFile(googleCredentials, getIdToken(googleCredentials));
 
     // We're not using pet SA key file (for security reasons), so auth is more complicated.
-    writeAdcCredentialFiles();
-    writeGsUtilCredentialFile();
+    if (gsutilLogin) {
```

Comment:
gsutilLogin -> writeGcloudAuthFiles (these are used by things other than gsutil eg bq)

// Only PassthroughApps.java may set to this to true. Otherwise, things get clobbered when tests are running in parallel on different cores.